### DISCUSSION TITLE
Possible an error with no-called Pubsub.init()

### DISCUSSION THREAD

**Discussion by Kaspiman** [redis/go-redis#2349] [Q&A]
Hello! 

I'm newbie in Go. While reading sources i found no-called Pubsub.init() method. Maybe Pubsub.exit channel never inits.

What do you think?

https://github.com/go-redis/redis/blob/09fcdbe6e494550a116ff9cdfb0d47057ea7d2ab/pubsub.go#L43

**Comment by vmihailenco** [comment]
Init is called [here](https://github.com/go-redis/redis/blob/master/redis.go#L678).
